userbot/__main__.py

```diff
@@ -94,6 +124,9 @@ def loaded_modules(self) -> int:
             quit(1)
         load_modules_count = modules.loaded_modules()
         sum_modules = len(ALL_MODULES)
+        log.info("Installing dependencies")
+        suc = installDeps()
+        log.info(f"Installed {suc}/{len(MODULE_DEPS)} pip packages")
```

Comment:
line 128-129 will always try to install the pip packages and print it's result each time HyperUBot gets booted also if the required packages are available already. This could spam the terminal much if many modules need extra packages.